atomic_physics/tests/test_utils.py

```diff
@@ -1,275 +1,205 @@
 import unittest
 
+import scipy.constants as ct

```

Comment:
small nit pick: I think we've used `import as consts` in the rest of this package. Let's try to be consistent to avoid surprises.